conf/sg/singapore01.yml

```diff
@@ -206,3 +201,64 @@ years:
     date: '2018-12-25'
     names:
       en: Christmas Day
+  '2020':
+  - public_holiday: true
+    date: '2020-01-01'
+    names:
+      en: New Year's Day
+  - public_holiday: true
+    date: '2020-01-01'
+    names: 
+      en: New Year's Day
```

Comment:
Duplicate ✂️

Reply:
I'll fix it 👍 thanks Timbo

Reply:
Oh no... I added a test for this and found 142 failures 😂 

```
Uruguay (all) 2019-04-19 has exactly one holiday: ["Good Friday", "Landing of the 33 Pariots Day"] FAIL (0.00s)
Serbia (all) 2032-05-02 has exactly one holiday: ["Orthodox Easter", "May Day Holiday"] FAIL (0.00s)
Western Australia 2038-04-26 has exactly one holiday: ["Easter Monday", "Anzac Day"] FAIL (0.00s)
West Virginia 2038-12-31 has exactly one holiday: ["New Year's Day", "New Year's Eve"] FAIL (0.00s)
Bulgaria (all) 2017-12-24 has exactly one holiday: ["Christmas Eve", "Christmas Eve"] FAIL (0.00s)
China (all) 2039-10-02 has exactly one holiday: ["Mid-Autumn Festival", "National Day Holiday"] FAIL (0.00s)
China (all) 2039-10-03 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
China (all) 2028-10-01 has exactly one holiday: ["National Day", "Mid-Autumn Festival Holiday"] FAIL (0.00s)
Italy (all) 2038-04-25 has exactly one holiday: ["Easter Sunday", "Anniversary of Liberation"] FAIL (0.00s)
Dubai (all) 2017-11-30 has exactly one holiday: ["Mouloud (The Prophet's Birthday)", "Commemoration Day"] FAIL (0.00s)
Nelson 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
South Korea (all) 2025-05-05 has exactly one holiday: ["Children's Day", "Buddha's Birthday"] FAIL (0.00s)
Ukraine (all) 2027-05-03 has exactly one holiday: ["Orthodox Easter Monday", "Labor Day"] FAIL (0.00s)
Hawaii 2016-03-25 has exactly one holiday: ["Prince Jonah Kuhio Kalanianaole Day", "Good Friday"] FAIL (0.00s)
Hawaii 2027-03-26 has exactly one holiday: ["Prince Jonah Kuhio Kalanianaole Day", "Good Friday"] FAIL (0.00s)
Victoria 2038-04-25 has exactly one holiday: ["Easter Sunday", "Anzac Day"] FAIL (0.00s)
Colombia (all) 2030-07-01 has exactly one holiday: ["The Sacred Heart of Jesus", "Saint Peter and Saint Paul"] FAIL (0.00s)
Northern Ireland 2011-07-12 has exactly one holiday: ["Battle of the Boyne", "Battle of the Boyne"] FAIL (0.00s)
South Australia 2038-04-26 has exactly one holiday: ["Easter Monday", "Anzac Day"] FAIL (0.00s)
China (all) 2025-10-04 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
China (all) 2017-10-03 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
Northern Ireland 2011-03-17 has exactly one holiday: ["St. Patrick's Day", "St. Patrick's Day"] FAIL (0.00s)
Latvia (all) 2035-05-13 has exactly one holiday: ["Mother's day", "Whit Sunday"] FAIL (0.00s)
Western Australia 2022-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day"] FAIL (0.00s)
China (all) 2025-10-05 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
China (all) 2036-10-05 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
Bulgaria (all) 2017-09-06 has exactly one holiday: ["Unification Day", "Unification Day"] FAIL (0.00s)
United Arab Emirates (all) 2017-11-30 has exactly one holiday: ["Mouloud (The Prophet's Birthday)", "Commemoration Day"] FAIL (0.00s)
Saudi Arabia (all) 2018-06-15 has exactly one holiday: ["Eid-al-Fitr (End of Ramadan)", "Eid-al-Fitr Holiday", "Eid-al-Fitr Holiday"] FAIL (0.00s)
South Korea (all) 2039-10-03 has exactly one holiday: ["Midautumn Festival Holiday", "National Foundation Day"] FAIL (0.00s)
China (all) 2028-10-03 has exactly one holiday: ["Mid-Autumn Festival", "National Day Holiday"] FAIL (0.00s)
Colombia (all) 2025-06-30 has exactly one holiday: ["The Sacred Heart of Jesus", "Saint Peter and Saint Paul"] FAIL (0.00s)
Westland 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
Thailand (all) 2017-12-05 has exactly one holiday: ["King Bhumibol Adulyadej day", "Father's Day"] FAIL (0.00s)
Lithuania (all) 2011-05-01 has exactly one holiday: ["Labour Day", "Mother's day"] FAIL (0.00s)
Wisconsin 2024-01-01 has exactly one holiday: ["New Year's Eve Day", "New Year's Day"] FAIL (0.00s)
Western Australia 2011-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day"] FAIL (0.00s)
Denmark (all) 2017-06-05 has exactly one holiday: ["Constitution Day", "Whit Monday"] FAIL (0.00s)
Ukraine (all) 2032-05-03 has exactly one holiday: ["Orthodox Easter Monday", "Labor Day"] FAIL (0.00s)
Ukraine (all) 2016-05-02 has exactly one holiday: ["Orthodox Easter Monday", "Labor Day"] FAIL (0.00s)
Wisconsin 2018-01-01 has exactly one holiday: ["New Year's Eve Day", "New Year's Day"] FAIL (0.00s)
Taiwan (all) 2024-04-04 has exactly one holiday: ["Children's Day", "Tomb Sweeping Day"] FAIL (0.00s)
Wisconsin 2035-01-01 has exactly one holiday: ["New Year's Eve Day", "New Year's Day"] FAIL (0.00s)
Serbia (all) 2016-05-01 has exactly one holiday: ["Orthodox Easter", "May Day"] FAIL (0.00s)
Wisconsin 2029-01-01 has exactly one holiday: ["New Year's Eve Day", "New Year's Day"] FAIL (0.00s)
West Virginia 2038-12-24 has exactly one holiday: ["Christmas Day", "Christmas Eve"] FAIL (0.00s)
Lithuania (all) 2016-05-01 has exactly one holiday: ["Labour Day", "Mother's day"] FAIL (0.00s)
South Australia 2016-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day / Proclamation Day"] FAIL (0.00s)
Western Australia 2016-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day"] FAIL (0.00s)
Serbia (all) 2021-05-02 has exactly one holiday: ["Orthodox Easter", "May Day Holiday"] FAIL (0.00s)
Northern Territory 2038-04-26 has exactly one holiday: ["Easter Monday", "Anzac Day"] FAIL (0.00s)
West Virginia 2027-12-31 has exactly one holiday: ["New Year's Day", "New Year's Eve"] FAIL (0.00s)
West Virginia 2027-12-24 has exactly one holiday: ["Christmas Day", "Christmas Eve"] FAIL (0.00s)
Serbia (all) 2021-05-03 has exactly one holiday: ["Orthodox Easter Monday", "May Day Holiday"] FAIL (0.00s)
China (all) 2020-10-02 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
China (all) 2025-10-06 has exactly one holiday: ["Mid-Autumn Festival", "National Day Holiday"] FAIL (0.00s)
Denmark (all) 2022-06-05 has exactly one holiday: ["Constitution Day", "Whit Sunday"] FAIL (0.00s)
West Virginia 2032-12-24 has exactly one holiday: ["Christmas Day", "Christmas Eve"] FAIL (0.00s)
Romania (all) 2016-05-01 has exactly one holiday: ["Orthodox Easter Sunday", "Labour Day"] FAIL (0.00s)
Bulgaria (all) 2017-09-22 has exactly one holiday: ["Independence Day", "Independence Day"] FAIL (0.00s)
Hawke's Bay 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
China (all) 2017-10-02 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
Thailand (all) 2018-08-13 has exactly one holiday: ["The Queen's Birthday observed", "Mother's Day observed"] FAIL (0.00s)
Western Australia 2033-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day"] FAIL (0.00s)
Iran (all) 2019-06-05 has exactly one holiday: ["Khordad National Uprising", "Eid-e-Fitr"] FAIL (0.00s)
South Canterbury 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
Serbia (all) 2016-05-02 has exactly one holiday: ["Orthodox Easter Monday", "May Day Holiday"] FAIL (0.00s)
Marlborough 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
Norway (all) 2032-05-17 has exactly one holiday: ["Constitution Day", "Whit Monday"] FAIL (0.00s)
Haiti (all) 2019-05-01 has exactly one holiday: ["Labour and Agriculture Day", "National Labour Day"] FAIL (0.00s)
Norway (all) 2027-05-17 has exactly one holiday: ["Constitution Day", "Whit Monday"] FAIL (0.00s)
Thailand (all) 2017-08-11 has exactly one holiday: ["The Queen's Birthday observed", "Mother's Day observed"] FAIL (0.00s)
Hawaii 2032-03-26 has exactly one holiday: ["Prince Jonah Kuhio Kalanianaole Day", "Good Friday"] FAIL (0.00s)
Comunidad Valenciana 2015-04-03 has exactly one holiday: ["Good Thursday", "Good Friday"] FAIL (0.00s)
Portugal (all) 2038-04-25 has exactly one holiday: ["Easter Sunday", "Freedom Day"] FAIL (0.00s)
China (all) 2031-10-01 has exactly one holiday: ["Mid-Autumn Festival", "National Day"] FAIL (0.00s)
Canterbury 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
South Australia 2011-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day / Proclamation Day"] FAIL (0.00s)
Serbia (all) 2032-05-03 has exactly one holiday: ["Orthodox Easter Monday", "May Day Holiday"] FAIL (0.00s)
Thailand (all) 2018-12-05 has exactly one holiday: ["King Bhumibol Adulyadej day", "Father's Day"] FAIL (0.00s)
China (all) 2020-10-03 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
Bulgaria (all) 2017-12-26 has exactly one holiday: ["Second Day of Christmas", "Second Day of Christmas"] FAIL (0.00s)
Western Australia 2039-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day"] FAIL (0.00s)
Bulgaria (all) 2017-05-24 has exactly one holiday: ["Culture and Literacy Day", "Culture and Literacy Day"] FAIL (0.00s)
Auckland 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
Tasmania 2011-04-25 has exactly one holiday: ["Easter Monday", "Anzac Day"] FAIL (0.00s)
West Virginia 2032-12-31 has exactly one holiday: ["New Year's Day", "New Year's Eve"] FAIL (0.00s)
South Australia 2033-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day / Proclamation Day"] FAIL (0.00s)
South Australia 2039-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day / Proclamation Day"] FAIL (0.00s)
Singapore (all) 2020-01-01 has exactly one holiday: ["New Year's Day", "New Year's Day"] FAIL (0.00s)
Colombia (all) 2038-07-05 has exactly one holiday: ["The Sacred Heart of Jesus", "Saint Peter and Saint Paul"] FAIL (0.00s)
Thailand (all) 2018-01-01 has exactly one holiday: ["New Year's Day", "New Year's Day observed"] FAIL (0.00s)
China (all) 2017-10-04 has exactly one holiday: ["Mid-Autumn Festival", "National Day Holiday"] FAIL (0.00s)
Philippines (all) 2020-04-09 has exactly one holiday: ["Day of Valor", "Maundy Thursday"] FAIL (0.00s)
South Korea (all) 2036-10-03 has exactly one holiday: ["Midautumn Festival Holiday", "National Foundation Day"] FAIL (0.00s)
West Virginia 2021-12-31 has exactly one holiday: ["New Year's Day", "New Year's Eve"] FAIL (0.00s)
Venezuela 2019-04-19 has exactly one holiday: ["Good Friday", "Independence Declaration Day"] FAIL (0.00s)
China (all) 2020-10-01 has exactly one holiday: ["Mid-Autumn Festival", "National Day"] FAIL (0.00s)
Italy (all) 2011-04-25 has exactly one holiday: ["Easter Monday", "Anniversary of Liberation"] FAIL (0.00s)
Southland 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
China (all) 2028-10-02 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
Denmark (all) 2033-06-05 has exactly one holiday: ["Constitution Day", "Whit Sunday"] FAIL (0.00s)
South Australia 2011-04-25 has exactly one holiday: ["Easter Monday", "Anzac Day"] FAIL (0.00s)
Wellington 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
Thailand (all) 2018-08-12 has exactly one holiday: ["The Queen's Birthday", "Mother's Day"] FAIL (0.00s)
Chatham Islands 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
West Virginia 2021-12-24 has exactly one holiday: ["Christmas Day", "Christmas Eve"] FAIL (0.00s)
China (all) 2023-10-01 has exactly one holiday: ["National Day", "Mid-Autumn Festival Holiday"] FAIL (0.00s)
Iran (all) 2019-03-20 has exactly one holiday: ["Oli Nationalisation Day", "Imam Ali' Birthday"] FAIL (0.00s)
Lithuania (all) 2033-05-01 has exactly one holiday: ["Labour Day", "Mother's day"] FAIL (0.00s)
Australian Capital Territory 2011-04-25 has exactly one holiday: ["Easter Monday", "Anzac Day"] FAIL (0.00s)
Otago 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
Lithuania (all) 2022-05-01 has exactly one holiday: ["Labour Day", "Mother's day"] FAIL (0.00s)
Taiwan (all) 2027-04-05 has exactly one holiday: ["Children's Day", "Tomb Sweeping Day"] FAIL (0.00s)
Taiwan (all) 2028-04-04 has exactly one holiday: ["Children's Day", "Tomb Sweeping Day"] FAIL (0.00s)
Lithuania (all) 2039-05-01 has exactly one holiday: ["Labour Day", "Mother's day"] FAIL (0.00s)
Bulgaria (all) 2017-12-25 has exactly one holiday: ["Christmas Day", "Christmas Day"] FAIL (0.00s)
South Korea (all) 2028-10-03 has exactly one holiday: ["Midautumn Festival", "National Foundation Day"] FAIL (0.00s)
Serbia (all) 2027-05-03 has exactly one holiday: ["Orthodox Easter Monday", "May Day Holiday"] FAIL (0.00s)
Iran (all) 2020-10-08 has exactly one holiday: ["Arabeen", "Birthday of Prophet Muhammad and Imam Sadeq"] FAIL (0.00s)
Comunidad de Madrid 2016-05-02 has exactly one holiday: ["Labour Day", "Comunidad de Madrid's Day"] FAIL (0.00s)
New South Wales 2038-04-25 has exactly one holiday: ["Easter Sunday", "Anzac Day"] FAIL (0.00s)
Greece (all) 2018-12-26 has exactly one holiday: ["Synaxis of the Mother of God", "Synaxis of the Mother of God"] FAIL (0.00s)
China (all) 2036-10-04 has exactly one holiday: ["Mid-Autumn Festival", "National Day Holiday"] FAIL (0.00s)
China (all) 2039-10-01 has exactly one holiday: ["National Day", "Mid-Autumn Festival Holiday"] FAIL (0.00s)
Ukraine (all) 2021-05-03 has exactly one holiday: ["Orthodox Easter Monday", "Labor Day"] FAIL (0.00s)
Taranaki 2038-04-26 has exactly one holiday: ["Easter Monday", "ANZAC Day"] FAIL (0.00s)
Serbia (all) 2027-05-02 has exactly one holiday: ["Orthodox Easter", "May Day Holiday"] FAIL (0.00s)
Norway (all) 2012-05-17 has exactly one holiday: ["Constitution Day", "Ascension Day"] FAIL (0.00s)
Bulgaria (all) 2017-03-03 has exactly one holiday: ["Liberation Day", "Liberation Day"] FAIL (0.00s)
China (all) 2036-10-06 has exactly one holiday: ["Mid-Autumn Festival Holiday", "National Day Holiday"] FAIL (0.00s)
South Korea (all) 2017-10-03 has exactly one holiday: ["Midautumn Festival Holiday", "National Foundation Day"] FAIL (0.00s)
Colombia (all) 2019-07-01 has exactly one holiday: ["The Sacred Heart of Jesus", "Saint Peter and Saint Paul"] FAIL (0.00s)
Otago 2035-03-26 has exactly one holiday: ["Easter Monday", "Otago Anniversary Day"] FAIL (0.00s)
South Australia 2022-12-26 has exactly one holiday: ["Christmas Day", "Boxing Day / Proclamation Day"] FAIL (0.00s)
Denmark (all) 2028-06-05 has exactly one holiday: ["Constitution Day", "Whit Monday"] FAIL (0.00s)
Northern Territory 2011-04-25 has exactly one holiday: ["Easter Monday", "Anzac Day"] FAIL (0.00s)
Australian Capital Territory 2038-04-25 has exactly one holiday: ["Easter Sunday", "Anzac Day"] FAIL (0.00s)
Pakistan (all) 2019-09-10 has exactly one holiday: ["9th Day of Muharram", "10th Day of Muharram"] FAIL (0.00s)
Iceland (all) 2011-04-21 has exactly one holiday: ["Maundy Thursday", "First Day of Summer"] FAIL (0.00s)
Thailand (all) 2017-08-12 has exactly one holiday: ["The Queen's Birthday", "Mother's Day"] FAIL (0.00s)
Iceland (all) 2038-04-22 has exactly one holiday: ["Maundy Thursday", "First Day of Summer"] FAIL (0.00s)
```

Fixes incoming...